.github/CONTRIBUTING.txt

```diff
@@ -13,6 +13,8 @@ Here's the long and short of it:
 
       git clone https://github.com/your-username/scikit-image.git
 
+   * cd scikit-image
```

Comment:
Yes, you are right. I should change that. Would I have to make another pull request or I can make it within this pull request?